### ISSUE TITLE
By default deps migrations not generating migrations

### ISSUE THREAD

**Issue by nilsenj** [delatbabel/contacts#2]
[Illuminate\Database\QueryException]
  SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "categories" does not exist (SQL: alter table "category_company" add
  constraint category_company_category_id_foreign foreign key ("category_id") references "categories" ("id") on delete casca
  de on update cascade)

  [PDOException]
  SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "categories" does not exist

basically this means of course that db categories doesn't exist from 
package
delatbabel/nestedcategories
basically no new migrations from deps not found in database/migration

tried
php artisan vendor:publish
php artisan vendor:publish --tag=migrations --force

still nothing)
php artisan dump-autoload
then run previous commands
composer update 
then previous commands

still nothing)

**Comment by delatbabel** [comment]
OK I have changed so that we correctly register the lower service providers. Try pulling the latest version and try again.

**Comment by nilsenj** [comment]
thanks) I'l try to do it after work

**Comment by nilsenj** [comment]
Ok after doing all the stuff i tried db:seed
and no problems found)
but please don't forget to tell that we'd better to add seeders to main database seeder))
by the way I need to know the order of seeding just not to break the db.
Oh I see one ContactSeeder it includes every seeder about contacts)
But what about categories?

**Comment by delatbabel** [comment]
Thanks for the comments. I will close this for now. I have tagged the new release.

I added some comments to the README file about the seeders.
